src/seq_mv/csr_matop_device.c

```diff
@@ -2044,36 +2270,43 @@ hypre_CSRMatrixSortRowOutOfPlace(hypre_CSRMatrix *A)
    hypre_TFree(hypre_CSRMatrixSortedJ(A), HYPRE_MEMORY_DEVICE);
    hypre_TFree(hypre_CSRMatrixSortedData(A), HYPRE_MEMORY_DEVICE);
 
-   hypre_CSRMatrixSortedJ(A) = hypre_TAlloc(HYPRE_Int, nnzA, HYPRE_MEMORY_DEVICE);
+   hypre_CSRMatrixSortedJ(A)    = hypre_TAlloc(HYPRE_Int,     nnzA, HYPRE_MEMORY_DEVICE);
    hypre_CSRMatrixSortedData(A) = hypre_TAlloc(HYPRE_Complex, nnzA, HYPRE_MEMORY_DEVICE);
 
-   hypre_TMemcpy(hypre_CSRMatrixSortedJ(A), A_j, HYPRE_Int, nnzA, HYPRE_MEMORY_DEVICE,
-                 HYPRE_MEMORY_DEVICE);
-   hypre_TMemcpy(hypre_CSRMatrixSortedData(A), A_a, HYPRE_Complex, nnzA, HYPRE_MEMORY_DEVICE,
-                 HYPRE_MEMORY_DEVICE);
+   hypre_TMemcpy(hypre_CSRMatrixSortedJ(A), A_j, HYPRE_Int, nnzA,
+                 HYPRE_MEMORY_DEVICE, HYPRE_MEMORY_DEVICE);
+   hypre_TMemcpy(hypre_CSRMatrixSortedData(A), A_a, HYPRE_Complex, nnzA,
+                 HYPRE_MEMORY_DEVICE, HYPRE_MEMORY_DEVICE);
 
    hypre_CSRMatrixJ(A) = hypre_CSRMatrixSortedJ(A);
    hypre_CSRMatrixData(A) = hypre_CSRMatrixSortedData(A);
 
    hypre_CSRMatrixSortRow(A);
 
-   hypre_CSRMatrixJ(A) = A_j;
+   hypre_CSRMatrixJ(A)    = A_j;
    hypre_CSRMatrixData(A) = A_a;
 
    return hypre_error_flag;
 }
-#endif
+#endif /* defined(HYPRE_USING_CUSPARSE) || defined(HYPRE_USING_ROCSPARSE) */
 
 #if defined(HYPRE_USING_CUSPARSE)
-/* @brief This functions sorts values and column indices in each row in ascending order INPLACE
- * @param[in] n Number of rows
- * @param[in] m Number of columns
- * @param[in] nnzA Number of nonzeroes
- * @param[in] *d_ia (Unsorted) Row indices
- * @param[in,out] *d_ja_sorted On Start: Unsorted column indices. On return: Sorted column indices
- * @param[in,out] *d_a_sorted On Start: Unsorted values. On Return: Sorted values corresponding with column indices
```

Comment:
I'm curious why this was changed? I believe "@param[]" is standard syntax for documentation generating tools like oxygen?

Reply:
Yes, right. That's for doxygen. I can revert this change. I confess I'm not paying attention to using doxygen keywords throughout the code. It's used in very few places in hypre. @rfalgout should we consider using doxygen keywords when writing new functions?

Reply:
Thanks, Rob!

Reply:
Note that this is a non-user function, so the doxygen stuff isn't needed here.  Still something to discuss though.